server/meeting.go

```diff
@@ -28,9 +29,13 @@ func (p *Plugin) GetMeeting(channelID string) (*Meeting, error) {
 		}
 	} else {
 		//Return a default value
+		channel, err := p.API.GetChannel(channelID)
+		if err != nil {
+			return nil, err
+		}
 		meeting = &Meeting{
 			Schedule:      time.Thursday,
-			HashtagFormat: "Jan02",
+			HashtagFormat: strings.Join([]string{fmt.Sprintf("%.15s", channel.Name), "Jan02"}, "-"),
```

Comment:
Please review the approach I took to this. It is a breaking change.

Reply:
@will7200 what is the behavior with older hashtag formats after you upgrade with these changes? It would just not recognize the date portion because of the missing {{}} correct?

Reply:
@marianunez Yes essentially. This PR will not support older hashtags. We can provide support for older hashtags, but that defaults the purpose of what is being accomplished with this fix.